internal/config/envvars.go

```diff
@@ -0,0 +1,11 @@
+package config
+
+
+const (
```

Comment:
Why not have the envvars defined in a separate file? They are used by provider and sweeper code; and are obviously unreleated to code in config.go. I would consider a separate package (`internal/envvars`) but since it's configuration related, I dropped it here.

The envvars were moved to `internal` because sweeping in the acceptance tests for the ssh_key uses them, and can't use them from `equinix/` because of import cycyle. If I wouldn't actually try to make acceptance tests of moved metal_ssh_key work, I wouldn't know that they need to be moved. I just spend 15 minutes researching on how to justify this :(.

Reply:
If they're going to the `config` package (which is IMO reasonable since they are used as configuration), then there's not really a compelling reason to add them to a separate file that only defines the constants: it's still in the `config` package and it still gets imported along with all of the code in `config.go`.  Moving the constants to a different file in the same package shouldn't lead to an import cycle.

Reply:
OK, I moved the const block to config.go and removed envvars.go.